base/src/main/java/org/aya/tyck/pat/PatTycker.java

```diff
@@ -320,7 +333,8 @@ yield doTyck(new Pattern.FakeShapedList(
   public @NotNull Tuple2<SeqView<Pat>, Term>
   visitPatterns(@NotNull Def.Signature sig, @NotNull SeqView<Pattern> stream, @Nullable Pattern outerPattern) {
     var results = MutableList.<Pat>create();
-    if (sig.param().isEmpty() && stream.isEmpty()) return Tuple.of(results.view(), sig.result());
+    // this path seems useless
```

Comment:
Is it still reachable?

Reply:
I mean like does the code coverage shows hit for this line

Reply:
It is reachable, but the behavior won't change if we delete that line:

* `sig.param().isEmpty`: won't go through the `while` loop
* `stream.isEmpty()`: won't go through the `if` test

Reply:
Oh